### PR TITLE
Fix event-rebuilder job

### PR DESCRIPTION
**Pull request by kruai**
# Overview

This PR is being created to address [ESSNTL-3367](https://issues.redhat.com/browse/ESSNTL-3367).
This will hopefully fix the issue with `events_topic_rebuilder.py` in Stage. This config change worked in ephemeral environments; I've also added debug statements to provide more insight when run with LOGLEVEL=DEBUG.

## PR Checklist

- [x] Keep PR title short, ideally under 72 characters
- [ ] Descriptive comments provided in complex code blocks
- [ ] Tests: validate optimal/expected output
- [ ] Tests: validate exceptions and failure scenarios
- [ ] Tests: edge cases
- [ ] Recovers or fails gracefully during potential resource outages (e.g. DB, Kafka)
- [ ] Uses [type hinting](https://docs.python.org/3/library/typing.html), if convenient
- [ ] Documentation, if this PR changes the way other services interact with host inventory
- [ ] Links to related PRs

## Secure Coding Practices Documentation Reference

You can find documentation on this checklist [here](https://github.com/RedHatInsights/secure-coding-checklist).

## Secure Coding Checklist

- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [ ] General Coding Practices

### CODE REVIEW

**Review by CodeHeeler — APPROVED** [review]
LGTM

File: app/config.py [hunk 1]
```diff
@@ -162,7 +162,7 @@ def __init__(self, runtime_environment):
         }
 
         self.events_kafka_consumer = {
-            "group_id": "inventory-events-rebuild",
```
Comment (FabriciaDinizRH) [inline app/config.py:165]: @kruai can you explain the changes to this file? Just so that I can understand things a little better.

**Review by FabriciaDinizRH — APPROVED** [review]

**Review by thearifismail — COMMENTED** [review]
`group_id` deals with partition assignment whereas `auto_offset_reset` tells where to look for when checking for messages to consume.  What led to choosing `auto_offset_reset` and removing `group_id`?

File: app/config.py (hunk 1 above)
Comment (kruai) [reply app/config.py:165]: Nope, sorry; I have no idea why this works. It didn't make sense that the consumer wasn't finding any messages, and I tried various solutions I found on StackOverflow :upside_down_face:

**Comment by kruai** [comment]
@thearifismail It was one of the suggestions in a StackOverflow solution, and it seemed to work. I'm afraid I have no more insight than that for this one

**Pull request merged**